playground/css-inject/__tests__/css-inject.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, test } from 'vitest'
+import { editFile, getColor, page, untilUpdated } from '~utils'
+
+test('css inject', async () => {
+  const linkedOutside = await page.$('.linked.outside')
+  const linkedInside = await page.$('.linked.inside')
+  const importedOutside = await page.$('.imported.outside')
+  const importedInside = await page.$('.imported.inside')
+
+  expect(await getColor(linkedOutside)).toBe('red')
+  expect(await getColor(linkedInside)).toBe('black')
+  expect(await getColor(importedOutside)).toBe('black')
```

Comment:
All of the tests are failing on this line

Reply:
They fail for production mode which is not in our scope. One would need to find another way for production mode support. As we worked around this issue I'll close this PR. Anyone is welcome to re-use my work to continue on this.